### PR TITLE
Post serve fake data

### PR DESCRIPTION
**Pull request by mukobi**
Make the posts service serve a fake post at the '/v1/' GET endpoint (get all posts) for testing/demoing.

### CODE REVIEW

**Comment by evankanderson** [comment]
@cmei4444 -- check that the supplied schema works for you.

**Comment by mukobi** [comment]
@cmei4444 does this schema work for you?

File: posts/app.py [hunk 1]
```diff
@@ -41,6 +41,17 @@ def upload_new_post():
 @app.route('/v1/', methods=['GET'])
 def get_all_posts():
     """Get all posts for the whole event."""
+    # TODO(mukobi) change from fake data to real query from db
+    return jsonify({
```
[the remark this answers is not in the document]
Comment (mukobi) [reply posts/app.py]: Whoops, looks like I forgot a post_id. Are you okay with changing "author" to "author_id" in the design doc so all the ID fields end in "_id" for consistency?

[the remark this answers is not in the document]
Comment (cmei4444) [reply posts/app.py]: Yeah that sounds good, can you update the design doc to reflect your changes?

[the remark this answers is not in the document]
Comment (mukobi) [reply posts/app.py]: Done

**Pull request merged**